### PR TITLE
0.2/121 tidy accordion code, fix scrolling issue

### PR DESCRIPTION
**Pull request by gingerbreadpup**

### CODE REVIEW

File: src/pages/Account.vue [hunk 1]
```diff
@@ -1,14 +1,102 @@
...
+                                <div>Voucher total</div>
+                                <div>Amount</div>
+                            </div>
+                        </label>
+                    </div>
 
-                <button>Log out</button>
+                    <!-- Tab row -->
+                    <div class="tab row">
+                        <input id="tab-two" type="checkbox" name="tabs">
```
Comment (charlesstrange2) [inline src/pages/Account.vue:23]: There's going to be a lot of these tabs when we alter it to generate autmagically. when we do that this is likely to be "tab-2" . Is that going to work for you?

**Review by charlesstrange2 — COMMENTED** [review]
tab names will probably be sequentially serialised. that ok? (see comment)

File: src/pages/Account.vue (hunk 1 above)
Comment (gingerbreadpup) [reply src/pages/Account.vue:23]: yeah, that's fine - you can use anything you want. :+1:

**Review by charlesstrange2 — APPROVED** [review]
:shipit:

**Pull request merged**